components/Auth/AuthProvider.tsx

```diff
@@ -53,7 +53,9 @@ export const AuthProvider: React.FC<{ children: ReactNode }> = ({
       }
     })
     return () => {
-      userDocUnsub()
+      if (typeof userDocUnsub === 'function') {
+        userDocUnsub()
+      }
```

Comment:
This is necessary because there are instances when we call userDocUnsub() and it is not defined and causes Next.js to throw an error.

Reply:
Weird - any idea where why? Also, I think you can optional chain function calls just in case the function is undefined, i.e. `useDocUnsub?()`

Reply:
Not a clue

Reply:
🤷🏼‍♂️